### PR TITLE
Replace jest with test:node + add playwright

### PR DESCRIPTION
**Pull request by sverhoeven**
Coverage did work as expected
`node --import tsx --test --experimental-test-coverage  src/*.test.ts` Prints coverage in js land and gives error when run with pnpm

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +55,24 @@ pnpm exec biome --help
...
+```shell
+pnpm test
+```
+
+The end-to-end tests are written with [playwright](https://playwright.dev/).
+The tests are in `apps/class-solid/tests/*.spec.ts` and can be run with the following command:
+
+```shell
+cd ./apps/class-solid
+pnpm exec playwright test
```
Comment (Peter9192) [inline README.md:73]: I had to `pnpm exec playwright install` first

Comment (Peter9192) [inline README.md:73]: Does it make sense to add a `test` command in `package.json` for this?

**Review by Peter9192 — APPROVED** [review]
Thanks for porting all my tests.. (-:

> `node --import tsx --test --experimental-test-coverage  src/*.test.ts` 

This gives me: `Warning: Could not report code coverage. TypeError: Cannot read properties of undefined (reading 'line')`

File: README.md (hunk 1 above)
Comment (sverhoeven) [reply README.md:82]: Did it tell you do that in a clear way? If not the docs should be improved.

Comment (sverhoeven) [reply README.md:82]: I also thought about it, but when running `pnpm test` from root the report command it prints is incorrect as it needs to be run with `--filter ./apps/class-solid` or`cd ./apps/class-solid`` before.

Also i want to distinguish between unit test vs end-to-end test

**Review by sverhoeven — COMMENTED** [review]
No problem porting.

Strange for me coverage gives following

```
node --import tsx --test --experimental-test-coverage  src/*.test.ts
...
ℹ start of coverage report
ℹ ------------------------------------------------------------------
ℹ file              | line % | branch % | funcs % | uncovered lines
ℹ ------------------------------------------------------------------
ℹ src/class.test.ts | 100.00 |   100.00 |  100.00 | 
ℹ src/class.ts      |  99.34 |    90.48 |   94.74 | 108
ℹ src/config.ts     |  98.98 |    66.67 |   33.33 | 98
ℹ src/runner.ts     |  70.83 |    80.00 |   66.67 | 7 9-12 14 24
ℹ ------------------------------------------------------------------
ℹ all files         |  97.14 |    88.89 |   86.67 |
ℹ ------------------------------------------------------------------
ℹ end of coverage report
...
```
and

```
node --import tsx --test --experimental-test-coverage --test-reporter=lcov --test-reporter-destination=lcov.info   src/*.test.ts
genhtml lcov.info --output-directory coverage
```
gives

_home_verhoes_git_classmodel_class-web_packages_class_coverage_src_runner ts gcov html

 Which looks like the lines covered in the transpiled js are different that ts code.

Could you try again with

```
node --version
v20.16.0
rm -rf node_modules/ */*/node_modules
pnpm install
cd packages/class
```

File: README.md (hunk 1 above)
Comment (Peter9192) [reply README.md:82]: It was quite clear, yes, but I would still prefer to update the docs like so:

```suggestion
```shell
cd ./apps/class-solid
pnpm exec playwright install  # first time only
pnpm exec playwright test
```

**Comment by Peter9192** [comment]
> Could you try again with
> 
> ```
> node --version
> v20.16.0
> rm -rf node_modules/ */*/node_modules
> pnpm install
> cd packages/class
> ```

I was on v20.11, switching to v20.16 and re-installing solved the issue. Thanks!

**Comment by sverhoeven** [comment]
> > Could you try again with
> > ```
> > node --version
> > v20.16.0
> > rm -rf node_modules/ */*/node_modules
> > pnpm install
> > cd packages/class
> > ```
> 
> I was on v20.11, switching to v20.16 and re-installing solved the issue. Thanks!

Nice, I added https://github.com/classmodel/class-web/blob/ee9d466e89c16591477ea613d017658750fdc542/packages/class/package.json#L24C2-L26C4 
which gives a warning during install. I can make it an errror by adding `enginesStrict:true`.

**Pull request merged**